native-sql-engine/cpp/src/CMakeLists.txt

```diff
@@ -13,6 +13,11 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(CMAKE_BUILD_TYPE  "Release")
 
+set(ARROW_ROOT "/usr/local")
```

Comment:
This is an upper/lower case issue, please try to use "lower cases in mvn" and "UPPER CASES in cpp".
it is a good finding, let me try if we can use the same cases in both mvn and cpp.

Reply:
Done to fix the issue for no default value in parameters.
The case sensitive usage for maven and cpp will be kept since coding style concern:
mvn -> lowercase, cpp -> UPPERCASE